plugins/providers/tableau/provider_test.go

```diff
@@ -24,6 +24,258 @@ func TestGetType(t *testing.T) {
 	})
 }
 
+func TestCreateConfig(t *testing.T) {
+
+	t.Run("should return error if there credentials are invalid", func(t *testing.T) {
+		providerURN := "test-provider-urn"
+		crypto := new(mocks.Crypto)
+		client := new(mocks.TableauClient)
+		p := tableau.NewProvider("", crypto)
+		p.Clients = map[string]tableau.TableauClient{
+			providerURN: client,
+		}
+
+		testcases := []struct {
+			pc *domain.ProviderConfig
+		}{
+			{ //invalid credentials struct
```

Comment:
let's use `name` in test case instead of comment

Reply:
added in the latest commit